frontend/src/components/Player.vue

```diff
@@ -70,6 +74,14 @@ export default {
       isHandSelectable: this.player.isHuman
     };
   },
+  computed: {
+    winner: function() {
+      return (
+        this.player.game.currentTrick.winner() == this.player &&
+        this.player.game.currentTrick.isFinished()
```

Comment:
Just saw this. Including `currentTrick.isFinished()` here changes the behavior we currently have in place. Right now, we show the winner of a trick even if a trick is not yet finished. The reasoning behind this is that it makes it easier to determine if you should play a high-value card (because your partner is winning the trick) or trump the trick (if your opponent is winning). I didn't want to change this blindly without bringing it up with you first. What do you think?

Reply:
I changed this deliberately for these reasons.

1. Movement of the Badge: When it was wrapped to the next line it was causing a lot of movement, due to the fast `playCard` action. I got a lot of negative feedback for this behaviour, because it's distracting and adds a lot of 'restlessness'.
2. Tradeoff between helping the player and beeing an 'unfair' asset. I strongly believe that keeping cards in memory and knowing who is leading the trick is something the player should be able to do on his own.
3. With the new trick design in mind, it should be really easy to distinguish who is winning the trick. We end up again with arguments one and two.

I'm not completely against showing it all the time, but I strongly believe it doesn't add enough value, instead it encourages not to care about the trick and adding stress (distraction + restlessness) to the player.